WordPress/src/main/java/org/wordpress/android/ui/CollapseFullScreenDialogFragment.java

```diff
@@ -146,12 +148,14 @@ public void setConfirmEnabled(boolean enabled) {
     public Dialog onCreateDialog(Bundle savedInstanceState) {
         initBuilderArguments();
 
-        Dialog dialog = new Dialog(requireContext(), getTheme()) {
```

Comment:
Minor (🔍): Consider making the `(ComponentDialog)` cast here instead of doing that further below just to get access to `getOnBackPressedDispatcher()`.

```
ComponentDialog dialog = (ComponentDialog) super.onCreateDialog(savedInstanceState);
```

PS: Same applies for the `FullScreenDialogFragment` change.

Reply:
Done: 1749865ed8a954c7dda33e64f2ea430cf5e93458

Reply:
Many thanks for this change @irfano ! 🥇